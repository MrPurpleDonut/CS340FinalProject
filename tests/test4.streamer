list<text> arrOfStrings = {"Hello", "there", "friends"};
