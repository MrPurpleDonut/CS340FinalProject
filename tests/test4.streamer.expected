Program parsed correctly!
---------------------------
--------Printing AST-------
---------------------------
Program
    Function List
    Statement List
        Declaration statement: arrOfStrings
            Type: list
                Type: text
            Assignment statement: arrOfStrings
                Expression List
                    Quoted string: "Hello"
                    Quoted string: "there"
                    Quoted string: "friends"
